packages/gitbook/src/components/Header/Dropdown.tsx

```diff
@@ -10,18 +10,22 @@ export type DropdownButtonProps<E extends HTMLElement = HTMLElement> = Omit<
     'ref'
 >;
 
-/**
- * Button with a dropdown.
- */
-export function Dropdown<E extends HTMLElement>(props: {
+export type DropdownProps<E extends HTMLElement = HTMLElement> = {
     /** Content of the button */
     button: (buttonProps: DropdownButtonProps<E>) => React.ReactNode;
     /** Content of the dropdown */
     children: React.ReactNode;
     /** Custom styles */
     className?: ClassValue;
-}) {
-    const { button, children, className } = props;
+    /** Whether to use fixed positioning (for fixed navbars) */
+    useFixedPosition?: boolean;
```

Comment:
I am not sure to understand why we need a special prop for this. The default behaviour of the dropdown is it's fixed and anchored to it's parent no matter what the situation is. Exposing a prop like this leaks implementation details that is not very clear for the caller of this API